src/modules/content-tree/content.tree.module.js

```diff
@@ -1,91 +1,95 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import List from './components/list/list.component';
+import ContentTree from './components/content-tree/content.tree';
+import { loadLocationItems } from './services/content.tree.service';
 
-const CLASS_IS_TREE_RESIZING = 'ez-is-tree-resizing';
+const findItem = (items, path) => {
```

Comment:
Why this method is outside the component?

Reply:
I'm not sure, maybe because it does not use `this`? We would have to ask Piotr. I will move this to the class body.